task14.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+if __name__ == "__main__":
+    one = np.arange(4)
```

Comment:
Variable names should be expressive and not generic. 
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0